inst/components/scss/toolbar.scss

```diff
@@ -0,0 +1,41 @@
+/* Toolbar */
+.bslib-toolbar {
+    display: flex;
+    align-items: center;
+
+    /* ---- Toolbar options ---- */
+
+    &[data-align="left"] {
+        margin-right: auto;
+    }
+
+    &[data-align="right"] {
+        margin-left: auto;
+    }
+
+    /* ---- Adjustments to other elements ---- */
+
+    // Ensures uniformity of font sizing in elements and sub-elements (e.g., input select)    
+    &,
+    & * {
+        font-size: 0.8rem;
+    }
+
+    & > * {
+        margin-bottom: 0 !important;
+        width: auto;
+        align-self: center;
+    }
+
+    // Ensures that inputs inherit the formatting of the toolbar and align correctly (e.g. input select)
+    > .form-group {
+        width: auto;
+    }
+
+    // Card header is flex by default, but card footer is not and must be in order for
+    // toolbar alignment to work
+    .card-footer:has(> &) {
+        display: flex;
+        align-items: center;
+    }
```

Comment:
```suggestion
```

Reply:
This, plus the `justify-content: end/start` `[data-align]` rules will align toolbar items internally left or right but we can keep the toolbar on its own row in the footer